crates/openapi/src/routes/profile.rs

```diff
@@ -1,23 +1,25 @@
+// ******************************************** V1 profile routes ******************************************** //
+
 #[cfg(feature = "v1")]
-/// Business Profile - Create
+/// Profile - Create
 ///
-/// Creates a new *business profile* for a merchant
+/// Creates a new *profile* for a merchant
 #[utoipa::path(
     post,
     path = "/account/{account_id}/business_profile",
```

Comment:
do we want to move the api to profile as well?  (in a backwards compatible manner)

Reply:
yes